amads/melody/contour/interpolation_contour.py

```diff
@@ -0,0 +1,272 @@
+"""
+Calculates the Interpolation Contour of a melody, along with related features, as
+implemented in the FANTASTIC toolbox of Müllensiefen (2009).
+"""
+
+__author__ = "David Whyatt"
+
+import numpy as np
+
+
+class InterpolationContour:
+    """Class for calculating and analyzing interpolation contours of melodies."""
+
+    def __init__(self, times: list[float], pitches: list[int]):
```

Comment:
Try and be consistent across your modules in whether times comes first or pitches. I think I would probably do pitch first, then time?

Reply:
I don't want to shirk responsibility, but that's how it's written in R! I agree that consistency would be good to have though